modules/criteoBidAdapter.js

```diff
@@ -471,10 +471,7 @@ export function tryGetCriteoFastBid() {
 
         if (verify(publisherTag, publisherTagHash, FAST_BID_PUBKEY_N, FAST_BID_PUBKEY_E)) {
           utils.logInfo('Using Criteo FastBid');
-          const script = document.createElement('script');
-          script.type = 'text/javascript';
-          script.text = publisherTag;
-          utils.insertElement(script);
+          eval(publisherTag); // eslint-disable-line no-eval
```

Comment:
Hi, after talking with the Prebid Org team we do not allow `eval` to be run by adapters.

no-eval is purposefully set in the lint check to keep this from happening.

I see that someone let this be merged sometime before, but unfortunately, they should not have allowed this in the first place.

Please update this PR to refrain from using Eval or close it.

Sorry for the inconvenience.

Reply:
Hello @robertrmartinez 

Following the discussion on the prebid's slack, are you ok to merge this ?

Thanks